src/mock/article.js

```diff
@@ -0,0 +1,459 @@
+const ARTICLE = {
+  article: {
+    title: ['lorem ipsum dolor sit amet'],
+    body: [`
+    Lorem
+    ipsum
```

Comment:
Why is every word placed on its own line?
This is not ideal at all

Reply:
let me have this fixed